R/bind_pdf.r

```diff
@@ -0,0 +1,32 @@
+#' Binds together the PDFs from an array job - is called by 'pdf_array_job()'. 
+#'
+bind_pdfs = function(temp_out_dir, final_out_dir, write_file_name){
+  require(grid)
+  require(gridExtra)
+  require(png)
```

Comment:
These need to be removed and **either** one of two sets of changes made

* "better" way - slightly more work
  * Add entries for each of these in **Imports** in `DESCRIPTION`. Add in alphabetical order (eases long-term maintenance)
  * namespace related calls. This is the tricker part. You need to add `PKGNAME::` to all the function calls so e.g., `readPNG` probably becomes `png::readPNG`
* "worse" way - slightly less work
  * add a `Depends` section to `DESCRIPTION`. **The last entry must not have a trailing comma**. See [this example Depends section](https://stash.ihme.washington.edu/users/miker985/repos/lbd.mbg/browse/DESCRIPTION#24-95)
  * That's it - "Depends" is like `library()`

Reply:
Modified in "better" way - thanks.